pkg/orchestration/wiring/saml/plugin.go

```diff
@@ -0,0 +1,97 @@
+package saml
+
+import (
+	"encoding/json"
+	"fmt"
+
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	"github.com/atlassian/voyager"
+	orch_v1 "github.com/atlassian/voyager/pkg/apis/orchestration/v1"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringutil/knownshapes"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringutil/svccatentangler"
+	"github.com/pkg/errors"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	clusterServiceClassExternalID                      = "277d87d1-4cf6-4d96-8bd6-3affb551f21c"
+	clusterServicePlanExternalID                       = "55beb5f4-16f0-4fe5-abcb-95e077271bde"
+	ResourceType                  voyager.ResourceType = "SAML"
+	ResourcePrefix                                     = "SAML"
+)
+
+type autowiringOnlySpec struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	URL         string `json:"url"`
+}
+type finalSpec struct {
+	autowiringOnlySpec
+}
+type WiringPlugin struct {
+	svccatentangler.SvcCatEntangler
```

Comment:
We are migrating from SvcCatEntangler to new helper functions, see #157, #158, #159, #160